calour/__init__.py

```diff
@@ -8,3 +8,8 @@
 
 __credits__ = "https://github.com/biocore/calour/graphs/contributors"
 __version__ = "0.1.0.dev0"
+
+from calour.experiment import *
+from calour.io import *
+from calour.filtering import *
+from calour.sorting import *
```

Comment:
with dynamically import functions into class methods, we only need to import the class to access them.